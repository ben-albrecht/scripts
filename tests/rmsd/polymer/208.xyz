20
0 2
 O        1.925834569     0.450721026    -1.243862838
 C        1.925590409     1.402189303    -2.249516325
 C        3.010594124    -0.009136111    -0.540599377
 H        0.309378068    -0.168919477    -5.372425496
Si       -0.142684063     0.816285101    -4.524727533
 C        0.982558758     1.118150956    -3.255914984
 O        2.655956393     2.391609406    -2.191296758
 H       -1.343712146     0.416678528    -3.985372698
 H       -0.271056480     2.003675778    -5.207137771
 O        3.698380519     0.970869140     0.108422041
 C        3.331051386     1.392915771     1.396336874
 H        4.040338352     2.204591870     1.682021266
 H        3.415123417     0.568371492     2.145337021
 H        2.291865075     1.796437539     1.439764487
 C        3.337981921    -1.325944063    -0.538525522
Si        2.660059599    -2.699402079    -1.429677059
 H        4.160493164    -1.620370614     0.124831113
 H        2.066459061    -2.307112354    -2.612445856
 H        1.714445060    -3.342275904    -0.658464602
 H        3.669082401    -3.590431225    -1.728749719
