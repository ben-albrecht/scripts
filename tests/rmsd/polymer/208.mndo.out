 *******************************************************************************
 ** Site#: 22060        For non-commercial use only     Version 15.038L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 15.038L web: HTTP://OpenMOPAC.net         Days remaining: 284    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                           MNDO CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Wed Apr 29 14:44:03 2015  *
 *  MNDO       - The MNDO Hamiltonian to be used
 *  PRECISE    - CRITERIA TO BE INCREASED BY 100 TIMES
 *  BFGS       - USE THE BFGS GEOMETRY OPTIMIZER
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
MNDO BFGS PRECISE
 comment line 1
 comment line 2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       O          1.84396000  *   0.36238000  *  -0.77963000  *
     2       C          1.66083000  *   1.34436000  *  -1.70122000  *
     3       C          3.05458000  *  -0.12575000  *  -0.39577000  *
     4       H          1.46138000  *  -0.03902000  *  -4.69280000  *
     5      Si          0.72205000  *   1.19697000  *  -4.39592000  *
     6       C          0.41704000  *   1.31802000  *  -2.54416000  *
     7       O          2.50938000  *   2.21269000  *  -1.86230000  *
     8       H         -0.56653000  *   1.18025000  *  -5.10383000  *
     9       H          1.50905000  *   2.35824000  *  -4.83729000  *
    10       O          3.92953000  *   0.75425000  *   0.17251000  *
    11       C          3.59621000  *   1.87363000  *   0.90986000  *
    12       H          4.52694000  *   2.38276000  *   1.23380000  *
    13       H          3.02243000  *   1.58225000  *   1.81544000  *
    14       H          3.00612000  *   2.58198000  *   0.29000000  *
    15       C          3.39226000  *  -1.42901000  *  -0.59567000  *
    16      Si          2.32435000  *  -2.69521000  *  -1.44475000  *
    17       H          4.36157000  *  -1.79406000  *  -0.27836000  *
    18       H          1.04355000  *  -2.16333000  *  -1.92822000  *
    19       H          2.06061000  *  -3.80227000  *  -0.51412000  *
    20       H          3.04979000  *  -3.21093000  *  -2.61479000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         O        1.8440    0.3624   -0.7796
     2         C        1.6608    1.3444   -1.7012
     3         C        3.0546   -0.1258   -0.3958
     4         H        1.4614   -0.0390   -4.6928
     5        Si        0.7221    1.1970   -4.3959
     6         C        0.4170    1.3180   -2.5442
     7         O        2.5094    2.2127   -1.8623
     8         H       -0.5665    1.1803   -5.1038
     9         H        1.5091    2.3582   -4.8373
    10         O        3.9295    0.7543    0.1725
    11         C        3.5962    1.8736    0.9099
    12         H        4.5269    2.3828    1.2338
    13         H        3.0224    1.5822    1.8154
    14         H        3.0061    2.5820    0.2900
    15         C        3.3923   -1.4290   -0.5957
    16        Si        2.3243   -2.6952   -1.4447
    17         H        4.3616   -1.7941   -0.2784
    18         H        1.0435   -2.1633   -1.9282
    19         H        2.0606   -3.8023   -0.5141
    20         H        3.0498   -3.2109   -2.6148

     ===================== shared-memory parallel CPU Section ======================
     **                                                                           **
     **  MOPAC calculations will run in shared-memory parallel CPUs               **
     **                                                                           **
     **  Number of Threads =  4                                                   **
     **                                                                           **
     **                                                                           **
     **  For information on this work, see:                                       **
     **                                                                           **
     **  Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.        **
     **        DOI: 10.1021/ct3004645.                                            **
     **                                                                           **
     ===================== shared-memory parallel CPU Section ======================
      


  H: (MNDO): M.J.S. DEWAR, W. THIEL, J. AM. CHEM. SOC., 99, 4899, (1977)
  C: (MNDO): M.J.S. DEWAR, W. THIEL, J. AM. CHEM. SOC., 99, 4899, (1977)
  O: (MNDO): M.J.S. DEWAR, W. THIEL, J. AM. CHEM. SOC., 99, 4899, (1977)
 Si: (MNDO): M.J.S.DEWAR, ET. AL. ORGANOMETALLICS  5, 375 (1986)

           Empirical Formula: C5 H10 O3 Si2  =    20 atoms



      MOLECULAR POINT GROUP   :   C1  


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =   28

      Geometry optimization using BFGS
 CYCLE:     1 TIME:   0.020 TIME LEFT:  2.00D  GRAD.:   146.218 HEAT: -4.500577    
 CYCLE:     2 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:    78.445 HEAT: -8.936602    
 CYCLE:     3 TIME:   0.039 TIME LEFT:  2.00D  GRAD.:    44.184 HEAT: -11.24927    
 CYCLE:     4 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:    52.940 HEAT: -12.05534    
 CYCLE:     5 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:    56.667 HEAT: -12.38947    
 CYCLE:     6 TIME:   0.020 TIME LEFT:  2.00D  GRAD.:    36.095 HEAT: -14.24836    
 CYCLE:     7 TIME:   0.273 TIME LEFT:  2.00D  GRAD.:    40.360 HEAT: -15.72241    
 CYCLE:     8 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:    54.340 HEAT: -17.46598    
 CYCLE:     9 TIME:   0.035 TIME LEFT:  2.00D  GRAD.:    49.417 HEAT: -19.50635    
 CYCLE:    10 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:    61.986 HEAT: -20.52149    
 CYCLE:    11 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:    52.218 HEAT: -22.38499    
 CYCLE:    12 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:    40.850 HEAT: -23.32813    
 CYCLE:    13 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:    31.598 HEAT: -24.32947    
 CYCLE:    14 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:    26.310 HEAT: -25.24015    
 CYCLE:    15 TIME:   0.020 TIME LEFT:  2.00D  GRAD.:    31.191 HEAT: -25.82145    
 CYCLE:    16 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:    26.608 HEAT: -26.42174    
 CYCLE:    17 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:    20.241 HEAT: -26.93436    
 CYCLE:    18 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:    26.658 HEAT: -27.80851    
 CYCLE:    19 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:    26.164 HEAT: -28.26445    
 CYCLE:    20 TIME:   0.039 TIME LEFT:  2.00D  GRAD.:    26.966 HEAT: -28.78612    
 CYCLE:    21 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:    20.311 HEAT: -29.26127    
 CYCLE:    22 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:    24.295 HEAT: -29.63968    
 CYCLE:    23 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:    20.198 HEAT: -30.22596    
 CYCLE:    24 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:    18.649 HEAT: -30.74118    
 CYCLE:    25 TIME:   0.035 TIME LEFT:  2.00D  GRAD.:    19.582 HEAT: -31.00608    
 CYCLE:    26 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:    21.126 HEAT: -31.26693    
 CYCLE:    27 TIME:   0.043 TIME LEFT:  2.00D  GRAD.:    14.322 HEAT: -31.48745    
 CYCLE:    28 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     9.536 HEAT: -31.68092    
 CYCLE:    29 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     7.949 HEAT: -31.82344    
 CYCLE:    30 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     7.679 HEAT: -31.90127    
 CYCLE:    31 TIME:   0.035 TIME LEFT:  2.00D  GRAD.:     6.841 HEAT: -31.97622    
 CYCLE:    32 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     5.818 HEAT: -32.02739    
 CYCLE:    33 TIME:   0.035 TIME LEFT:  2.00D  GRAD.:     5.524 HEAT: -32.06699    
 CYCLE:    34 TIME:   0.051 TIME LEFT:  2.00D  GRAD.:     4.905 HEAT: -32.10450    
 CYCLE:    35 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     3.850 HEAT: -32.14821    
 CYCLE:    36 TIME:   0.035 TIME LEFT:  2.00D  GRAD.:     4.001 HEAT: -32.17891    
 CYCLE:    37 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     4.400 HEAT: -32.20205    
 CYCLE:    38 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     4.300 HEAT: -32.22397    
 CYCLE:    39 TIME:   0.039 TIME LEFT:  2.00D  GRAD.:     3.563 HEAT: -32.24647    
 CYCLE:    40 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     3.443 HEAT: -32.27286    
 CYCLE:    41 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     4.330 HEAT: -32.29969    
 CYCLE:    42 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     5.072 HEAT: -32.32876    
 CYCLE:    43 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     6.529 HEAT: -32.36089    
 CYCLE:    44 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     4.912 HEAT: -32.38335    
 CYCLE:    45 TIME:   0.039 TIME LEFT:  2.00D  GRAD.:     5.812 HEAT: -32.44321    
 CYCLE:    46 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     7.016 HEAT: -32.48723    
 CYCLE:    47 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     3.760 HEAT: -32.51313    
 CYCLE:    48 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     4.573 HEAT: -32.53240    
 CYCLE:    49 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     4.354 HEAT: -32.55883    
 CYCLE:    50 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     4.153 HEAT: -32.59149    
 CYCLE:    51 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     3.949 HEAT: -32.61194    
 CYCLE:    52 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     4.888 HEAT: -32.63256    
 CYCLE:    53 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     4.159 HEAT: -32.65071    
 CYCLE:    54 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     5.884 HEAT: -32.68073    
 CYCLE:    55 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     8.615 HEAT: -32.74510    
 CYCLE:    56 TIME:   0.035 TIME LEFT:  2.00D  GRAD.:     4.912 HEAT: -32.78196    
 CYCLE:    57 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     7.485 HEAT: -32.82935    
 CYCLE:    58 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     5.013 HEAT: -32.86114    
 CYCLE:    59 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     3.812 HEAT: -32.88502    
 CYCLE:    60 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     3.908 HEAT: -32.90215    
 CYCLE:    61 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     3.011 HEAT: -32.91609    
 CYCLE:    62 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     3.217 HEAT: -32.92753    
 CYCLE:    63 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     2.989 HEAT: -32.94018    
 CYCLE:    64 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     3.836 HEAT: -32.96016    
 CYCLE:    65 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     3.678 HEAT: -32.97530    
 CYCLE:    66 TIME:   0.039 TIME LEFT:  2.00D  GRAD.:     3.735 HEAT: -32.99327    
 CYCLE:    67 TIME:   0.133 TIME LEFT:  2.00D  GRAD.:     3.435 HEAT: -33.00660    
 CYCLE:    68 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     2.918 HEAT: -33.01980    
 CYCLE:    69 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     3.314 HEAT: -33.02776    
 CYCLE:    70 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     3.333 HEAT: -33.04267    
 CYCLE:    71 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     2.784 HEAT: -33.05281    
 CYCLE:    72 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     2.554 HEAT: -33.06560    
 CYCLE:    73 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     3.031 HEAT: -33.07328    
 CYCLE:    74 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     3.140 HEAT: -33.08708    
 CYCLE:    75 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     3.207 HEAT: -33.10030    
 CYCLE:    76 TIME:   0.035 TIME LEFT:  2.00D  GRAD.:     3.034 HEAT: -33.11314    
 CYCLE:    77 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     3.432 HEAT: -33.12268    
 CYCLE:    78 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     4.344 HEAT: -33.14391    
 CYCLE:    79 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     3.470 HEAT: -33.15827    
 CYCLE:    80 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     3.319 HEAT: -33.17498    
 CYCLE:    81 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:    37.111 HEAT: -33.19582    
 CYCLE:    82 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     3.072 HEAT: -33.19721    
 CYCLE:    83 TIME:   0.051 TIME LEFT:  2.00D  GRAD.:     2.432 HEAT: -33.20734    
 CYCLE:    84 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     2.240 HEAT: -33.21618    
 CYCLE:    85 TIME:   0.020 TIME LEFT:  2.00D  GRAD.:     1.766 HEAT: -33.22124    
 CYCLE:    86 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     2.087 HEAT: -33.22346    
 CYCLE:    87 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     1.610 HEAT: -33.22555    
 CYCLE:    88 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     1.913 HEAT: -33.23011    
 CYCLE:    89 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     2.125 HEAT: -33.23558    
 CYCLE:    90 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     2.157 HEAT: -33.24038    
 CYCLE:    91 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     1.936 HEAT: -33.24515    
 CYCLE:    92 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     1.801 HEAT: -33.24980    
 CYCLE:    93 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     2.383 HEAT: -33.25556    
 CYCLE:    94 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     3.503 HEAT: -33.26647    
 CYCLE:    95 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     2.458 HEAT: -33.28049    
 CYCLE:    96 TIME:   0.035 TIME LEFT:  2.00D  GRAD.:     3.561 HEAT: -33.29297    
 CYCLE:    97 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     2.366 HEAT: -33.29938    
 CYCLE:    98 TIME:   0.035 TIME LEFT:  2.00D  GRAD.:     1.615 HEAT: -33.30504    
 CYCLE:    99 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     1.971 HEAT: -33.30891    
 TEST ON GRADIENT SATISFIED
                    HOWEVER, A COMPONENT OF GRADIENT IS LARGER THAN  0.20

 CYCLE:   100 TIME:   0.035 TIME LEFT:  2.00D  GRAD.:     1.325 HEAT: -33.31166    
 TEST ON GRADIENT SATISFIED
                    HOWEVER, A COMPONENT OF GRADIENT IS LARGER THAN  0.20

 CYCLE:   101 TIME:   0.043 TIME LEFT:  2.00D  GRAD.:     1.199 HEAT: -33.31439    
 TEST ON GRADIENT SATISFIED
                    HOWEVER, A COMPONENT OF GRADIENT IS LARGER THAN  0.20

 CYCLE:   102 TIME:   0.047 TIME LEFT:  2.00D  GRAD.:     1.284 HEAT: -33.31536    
 TEST ON GRADIENT SATISFIED
                    HOWEVER, A COMPONENT OF GRADIENT IS LARGER THAN  0.20

 CYCLE:   103 TIME:   0.047 TIME LEFT:  2.00D  GRAD.:     1.193 HEAT: -33.31601    
 TEST ON GRADIENT SATISFIED
                    HOWEVER, A COMPONENT OF GRADIENT IS LARGER THAN  0.20

           THERE HAVE BEEN 3 ATTEMPTS TO REDUCE THE GRADIENT.
           DURING THESE ATTEMPTS THE ENERGY DROPPED BY LESS THAN 0.0 KCAL/MOLE
           FURTHER CALCULATION IS NOT JUSTIFIED AT THIS TIME.

 -------------------------------------------------------------------------------
MNDO BFGS PRECISE
 comment line 1
 comment line 2


     GRADIENT TEST NOT PASSED, BUT FURTHER WORK NOT JUSTIFIED 
     SCF FIELD WAS ACHIEVED                                   


                              MNDO CALCULATION
                                                       MOPAC2012 (Version: 15.038L)
                                                       Wed Apr 29 14:44:07 2015
                                                       No. of days left = 284




          FINAL HEAT OF FORMATION =        -33.31685 KCAL/MOL =    -139.39770 KJ/MOL


          TOTAL ENERGY            =      -1919.08963 EV
          ELECTRONIC ENERGY       =      -8491.46775 EV  POINT GROUP:     C1  
          CORE-CORE REPULSION     =       6572.37811 EV
          COSMO AREA              =        229.07 SQUARE ANGSTROMS
          COSMO VOLUME            =        240.87 CUBIC ANGSTROMS

          GRADIENT NORM           =          1.30949
          IONIZATION POTENTIAL    =          9.573790 EV
          HOMO LUMO ENERGIES (EV) =         -9.574 -1.946
          NO. OF FILLED LEVELS    =         28
          MOLECULAR WEIGHT        =        174.303

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    13    H     9     8.34907
            H    12    H    20     6.37486
            O     7    H     8     3.19698


          SCF CALCULATIONS        =        152
 
          WALL-CLOCK TIME         =          3.566 SECONDS
          COMPUTATION TIME        =          9.921 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       O          1.92583457  *   0.45072103  *  -1.24386284  *
     2       C          1.92559041  *   1.40218930  *  -2.24951633  *
     3       C          3.01059412  *  -0.00913611  *  -0.54059938  *
     4       H          0.30937807  *  -0.16891948  *  -5.37242550  *
     5      Si         -0.14268406  *   0.81628510  *  -4.52472753  *
     6       C          0.98255876  *   1.11815096  *  -3.25591498  *
     7       O          2.65595639  *   2.39160941  *  -2.19129676  *
     8       H         -1.34371215  *   0.41667853  *  -3.98537270  *
     9       H         -0.27105648  *   2.00367578  *  -5.20713777  *
    10       O          3.69838052  *   0.97086914  *   0.10842204  *
    11       C          3.33105139  *   1.39291577  *   1.39633687  *
    12       H          4.04033835  *   2.20459187  *   1.68202127  *
    13       H          3.41512342  *   0.56837149  *   2.14533702  *
    14       H          2.29186507  *   1.79643754  *   1.43976449  *
    15       C          3.33798192  *  -1.32594406  *  -0.53852552  *
    16      Si          2.66005960  *  -2.69940208  *  -1.42967706  *
    17       H          4.16049316  *  -1.62037061  *   0.12483111  *
    18       H          2.06645906  *  -2.30711235  *  -2.61244586  *
    19       H          1.71444506  *  -3.34227590  *  -0.65846460  *
    20       H          3.66908240  *  -3.59043123  *  -1.72874972  *

                             CARTESIAN COORDINATES

   1    O        1.925834569     0.450721026    -1.243862838
   2    C        1.925590409     1.402189303    -2.249516325
   3    C        3.010594124    -0.009136111    -0.540599377
   4    H        0.309378068    -0.168919477    -5.372425496
   5   Si       -0.142684063     0.816285101    -4.524727533
   6    C        0.982558758     1.118150956    -3.255914984
   7    O        2.655956393     2.391609406    -2.191296758
   8    H       -1.343712146     0.416678528    -3.985372698
   9    H       -0.271056480     2.003675778    -5.207137771
  10    O        3.698380519     0.970869140     0.108422041
  11    C        3.331051386     1.392915771     1.396336874
  12    H        4.040338352     2.204591870     1.682021266
  13    H        3.415123417     0.568371492     2.145337021
  14    H        2.291865075     1.796437539     1.439764487
  15    C        3.337981921    -1.325944063    -0.538525522
  16   Si        2.660059599    -2.699402079    -1.429677059
  17    H        4.160493164    -1.620370614     0.124831113
  18    H        2.066459061    -2.307112354    -2.612445856
  19    H        1.714445060    -3.342275904    -0.658464602
  20    H        3.669082401    -3.590431225    -1.728749719


           Empirical Formula: C5 H10 O3 Si2  =    20 atoms



      MOLECULAR POINT GROUP   :   C1  


                  EIGENVALUES  
 -46.09035 -42.95998 -39.86044 -33.59518 -30.90248 -28.49514 -24.01966 -21.56011
 -19.96951 -19.01077 -17.59971 -16.72575 -16.16557 -15.65005 -15.17740 -15.07367
 -13.96166 -13.44255 -12.65160 -12.57862 -12.38268 -12.10248 -12.03291 -11.80154
 -11.57258 -11.32954  -9.72190  -9.57379  -1.94573   0.41133   1.22019   2.18817
   2.25138   2.32506   2.56898   2.67754   2.79648   2.83268   3.04551   3.34859
   3.74570   3.92222   4.04968   4.53041   4.73897   4.94625   5.87448   6.25178
   6.87704   7.59019


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          O          -0.343804        6.3438     1.80003     4.54377
    2          C           0.421671        3.5783     1.17158     2.40675
    3          C           0.312900        3.6871     1.18716     2.49994
    4          H          -0.371914        1.3719     1.37191
    5         Si           1.606449        2.3936     0.97704     1.41651
    6          C          -0.394307        4.3943     1.44859     2.94572
    7          O          -0.362875        6.3629     1.89141     4.47147
    8          H          -0.365069        1.3651     1.36507
    9          H          -0.354139        1.3541     1.35414
   10          O          -0.305294        6.3053     1.80655     4.49874
   11          C           0.224591        3.7754     1.22385     2.55156
   12          H           0.020504        0.9795     0.97950
   13          H          -0.021171        1.0212     1.02117
   14          H          -0.011503        1.0115     1.01150
   15          C          -0.539947        4.5399     1.29197     3.24798
   16         Si           1.578457        2.4215     0.98514     1.43640
   17          H           0.065631        0.9344     0.93437
   18          H          -0.390668        1.3907     1.39067
   19          H          -0.384129        1.3841     1.38413
   20          H          -0.385383        1.3854     1.38538
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.    -2.998    -2.000    -1.670     3.972
 HYBRID         0.719     0.515     1.321     1.590
 SUM           -2.279    -1.484    -0.349     2.742


          ATOMIC ORBITAL ELECTRON POPULATIONS

    Atom      s        px        py        pz   
    1  O   1.80003   1.44652   1.60645   1.49081
    2  C   1.17158   0.82390   0.80138   0.78146
    3  C   1.18716   0.78448   0.84458   0.87087
    4  H   1.37191
    5 Si   0.97704   0.46327   0.49482   0.45843
    6  C   1.44859   1.38933   0.47211   1.08428
    7  O   1.89141   1.42287   1.29741   1.75119
    8  H   1.36507
    9  H   1.35414
   10  O   1.80655   1.61117   1.55983   1.32773
   11  C   1.22385   0.90360   0.89542   0.75254
   12  H   0.97950
   13  H   1.02117
   14  H   1.01150
   15  C   1.29197   1.09096   1.01098   1.14604
   16 Si   0.98514   0.47660   0.48402   0.47578
   17  H   0.93437
   18  H   1.39067
   19  H   1.38413
   20  H   1.38538

          JOB ENDED NORMALLY



 TOTAL JOB TIME:             3.68 SECONDS

 == MOPAC DONE ==
